examples/bevy_simple/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use bevy::prelude::*;
+use mun_runtime::Runtime as MunRuntime;
+
+const MUNLIB_PATH: &str = "mun/target/mod.munlib";
+
+fn main() {
+    App::new()
+        .add_plugins(MinimalPlugins)
+        .add_plugin(MunPlugin)
+        .run();
+}
+
+struct MunPlugin;
+
+impl Plugin for MunPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_startup_system(setup.exclusive_system())
```

Comment:
It would be nice to explain in the example why the `exclusive_system` is required.

Actually, in general it would be nice to have some comment in here that explains that the runtime is non-sync and that bevy has support for this through `exclusive_system`, `insert_non_send_resource`, and `NonSendMut`.

Reply:
Added additional comments.

Reply:
Perfect!